src/common/src/array/data_chunk.rs

```diff
@@ -42,20 +40,23 @@ use crate::util::value_encoding::{
 /// [`DataChunk`] is a collection of Columns,
 /// a with visibility mask for each row.
 /// For instance, we could have a [`DataChunk`] of this format.
+/// ```text
 /// | v1 | v2 | v3 |
 /// |----|----|----|
 /// | 1  | a  | t  |
 /// | 2  | b  | f  |
 /// | 3  | c  | t  |
 /// | 4  | d  | f  |
-///
+/// ```
 /// Our columns are v1, v2, v3.
```

Comment:
Isn't it a markdown table? But it doesn't look good in VS Code, so codeblock is good. 😄 

<img width="428" alt="image" src="https://user-images.githubusercontent.com/37948597/235666253-c22849cb-b16a-456d-8ca7-356920573f87.png">

Reply:
I added ``` because it looked pretty bad in rust docs. 😇

![WX20230503-015726@2x](https://user-images.githubusercontent.com/15158738/235746673-409263e7-2cfe-404f-810f-b0970076a430.png)